style.css

```diff
@@ -458,3 +470,121 @@
 hr {
   width: 100%;
 }
+
+.contact-form {
+  margin-top: 150px;
+  border-top-left-radius: 68px;
+  display: flex;
+  flex-direction: column;
+  width: 373px;
+  height: 779px;
+  background: #6070ff;
+  background-image: url("images/contact\ form.png");
+  background-size: 410px 740px;
+  background-repeat: no-repeat;
```

Comment:
- [OPTIONAL] Great work for implementing the required changes from the previous reviewers. However, there's little thing you still need to correct. The pixel values used for the background-size property for the background-image has distorted the bg image. For it to look exactly as we have it on the Figma design, you may need to use `cover` value for the background-size. I tried it with devTool and I was able to have it as we have it on the Figma design. Kindly look into it and fix.

### Your Design
![bg-img-logo](https://user-images.githubusercontent.com/26597432/220438968-d8012928-3464-45d0-9c37-c0e6a114acf6.png)

### Figma Design
![figma-design](https://user-images.githubusercontent.com/26597432/220439015-3d35c678-c87a-4f0c-b798-1bd7c2233e5a.png)


- [OPTIONAL] There's an extra white space by the right of your page. Kindly look into it and remove it as for your design to look like that of Figma design

[your-app-behaviour.webm](https://user-images.githubusercontent.com/26597432/220439305-35915ce0-5c0c-42fe-a2d2-410377f3ab7f.webm)